src/vs/workbench/contrib/preferences/common/preferencesContribution.ts

```diff
@@ -155,7 +155,7 @@ registry.registerConfiguration({
 				nls.localize('settingsSearchTocBehavior.hide', "Hide the Table of Contents while searching. The search results will not be grouped by category, and instead will be sorted by similarity to the query, with exact keyword matches coming first."),
 				nls.localize('settingsSearchTocBehavior.filter', "Filter the Table of Contents to just categories that have matching settings. Clicking a category will filter the results to that category. The search results will be grouped by category."),
 			],
-			'description': nls.localize('settingsSearchTocBehavior', "Controls the behavior of the settings editor Table of Contents while searching."),
+			'description': nls.localize('settingsSearchTocBehavior', "Controls the behavior of the Settings editor Table of Contents while searching. If this setting is being changed in the Settings editor, the setting will take effect after the search query is modified."),
```

Comment:
I think it should either be "Settings Editor" or "settings editor"?

Reply:
I'm using the same casing as the release notes, where I believe it's always been "Settings editor"
Adding @gregvanl to the thread to double-check

Reply:
Yep, we use "Settings editor" in documentation and release notes